### ISSUE TITLE
math/geometry text overlaps

### ISSUE THREAD

**Issue by Darksonn** [kth-competitive-programming/kactl#128]
I tried compiling the pdf using `make kactl` but the text in the math/geometry part of the pdf is broken as seen below. I am compiling on arch linux with pdflatex 3.14159265-2.6-1.40.20. 

![scr](https://user-images.githubusercontent.com/928074/67150847-37c66380-f2bd-11e9-8929-3e13a02e5423.png)

The table in 6.2.1 also overlaps with the title for section 6.2.2 binomial, but otherwise I don't see any issues.

**Comment by simonlindholm** [comment]
Yeah, somehow that part of the pdf is prone to screwing up when things change size e.g. because of different LaTeX packages. It's some bug in multicol that comes up in relation to illustrations, I think... but I'm not good enough with LaTeX to understand why or how to fix/work around it, so so far I've just tried to keep section sizes down to avoid the problem.

PRs accepted if you figure something out or find a reasonable work-around. :/ If it's just for local use you could cut some of the trigonometry section out.

**Comment by speedyguy17** [comment]
+1 in this exact same part, which I suspect is to do with the image. If I have some time at some point, I'll dig into it.

**Comment by Tyilo** [comment]
Although it doesn't seem to fix it completely, the following patch seems to improve it alot:

```
iff --git a/content/math/chapter.tex b/content/math/chapter.tex
index 6729585..800f13f 100644
--- a/content/math/chapter.tex
+++ b/content/math/chapter.tex
@@ -63,7 +63,9 @@ magic flux $F=b^2+d^2-a^2-c^2$:
 $ef = ac + bd$, and $A = \sqrt{(p-a)(p-b)(p-c)(p-d)}$.
 
 \subsection{Spherical coordinates}
-\centerline{\includegraphics[width=25mm]{content/math/sphericalCoordinates}}
+\begin{center}
+\includegraphics[width=25mm]{content/math/sphericalCoordinates}
+\end{center}
```

Result of `make kactl` from master before applying the patch:

![](https://i.imgur.com/kt6UEej.png)

and after:

![](https://i.imgur.com/NVMblHR.png)

**Comment by simonlindholm** [comment]
Thanks for the note, @Tyilo. I committed that change at 390bc1626e2ea6853a30e56289296fc2e12181bc. It seems to improve the display marginally on my system as well.

**Comment by khokho** [comment]
![image](https://user-images.githubusercontent.com/6597974/134704180-d31a6b05-a16f-458d-992f-5077b1a3022d.png)
I cannot seem to be able to fix this. The geometry section seems fine in my pdf. 
using:
`pdfTeX 3.141592653-2.6-1.40.22 (TeX Live 2021/Arch Linux)`

**Comment by khokho** [comment]
```
diff --git a/content/graph/chapter.tex b/content/graph/chapter.tex
index c51c641..c794df2 100644
--- a/content/graph/chapter.tex
+++ b/content/graph/chapter.tex
@@ -58,4 +58,4 @@
                % Source: https://en.wikipedia.org/wiki/Erd%C5%91s%E2%80%93Gallai_theorem
                % Test: stress-tests/graph/erdos-gallai.cpp
                A simple graph with node degrees $d_1 \ge \dots \ge d_n$ exists iff $d_1 + \dots + d_n$ is even and for every $k = 1\dots n$,
-               \[ \sum _{i=1}^{k}d_{i}\leq k(k-1)+\sum _{i=k+1}^{n}\min(d_{i},k). \]
+               \[ \smash{\sum _{i=1}^{k}d_{i}\leq k(k-1)+\sum _{i=k+1}^{n}\min(d_{i},k).} \]
```
